src/sentry/web/urls.py

```diff
@@ -89,11 +89,6 @@
     ]
 
 urlpatterns += [
-    url(
-        r"^api/(?P<project_id>[\w_-]+)/crossdomain\.xml$",
```

Comment:
Wait, stop, we actually shouldn't drop this. It still works:

```
$ http get https://sentry.io/api/1/crossdomain.xml
HTTP/1.1 200 OK
Connection: keep-alive
Content-Length: 544
Content-Type: application/xml
Date: Mon, 20 Jun 2022 19:12:02 GMT
Server: nginx
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
cache-control: max-age=60
content-language: en
vary: Accept-Language, Cookie
x-content-type-options: nosniff
x-envoy-attempt-count: 1
x-envoy-upstream-service-time: 8
x-frame-options: deny
x-served-by: getsentry-web-default-production-559b746867-m6g77
x-served-by: lb-11
x-xss-protection: 1; mode=block

<?xml version="1.0"?>
<!DOCTYPE cross-domain-policy SYSTEM
    "http://www.adobe.com/xml/dtds/cross-domain-policy.dtd">
<cross-domain-policy>

        <allow-access-from domain="app.getsentry.com" secure="false" />

        <allow-access-from domain="getsentry.com" secure="false" />

        <allow-access-from domain="www.getsentry.com" secure="false" />

        <allow-access-from domain="sentry.io" secure="false" />

    <allow-http-request-headers-from domain="*" headers="*" secure="false" />
</cross-domain-policy>
```

Reply:
I can't find resources online as to whether this or the `/crossdomain.xml` would be used, but we should check traffic for this endpoint before killing it

Reply:
Ah indeed, i hit the wrong URL. That makes more sense. Since we're deprecating legacy ingest anyways I'm assuming this will silently go anyways. There will most likely still be hits to `/crossdomain.xml` since a bunch of robots, malware and related tools like to crawl it.

Reply:
i would suggest we keep both endpoints as-is for now. I would also be interested in investigating whether we still get requests, but don't have the capability to do so.